README.md

```diff
@@ -12,7 +12,7 @@ High-performance spatio-temporal data analysis platform for OpenStreetMap full-h
 [![status: active](https://github.com/GIScience/badges/raw/master/status/active.svg)](https://github.com/GIScience/badges#active)
 [![LICENSE](https://img.shields.io/github/license/GIScience/oshdb)](COPYING)
 
-The OSHDB allows to investigate the evolution of the amount of data and the contributions to the OpenStreetMap project. It combines easy access to the historical OSM data with high querying performance. Use cases of the OSHDB include data quality analysis, computing of aggregated data statistics and OSM data extraction.
+The OSHDB allows to investigate the evolution of the amount of data, and the contributions to the OpenStreetMap project. It combines easy access to the historical OSM data with high querying performance. Use cases of the OSHDB include data quality analysis, computing of aggregated data statistics and OSM data extraction.
```

Comment:
I believe a serial comma would not be used when there are only two subjects to consider in the _and_ part of the sentence, would it?

Reply:
✔

Reply:
✔